### PR TITLE
Feature - New Security Researcher List and View Pages

### PR DESCRIPTION
**Pull request by chuckbergeron**
screenshot 2019-02-11 14 26 08
screenshot 2019-02-11 14 26 13
screenshot 2019-02-11 14 26 21
screenshot 2019-02-11 14 38 26
screenshot 2019-02-11 14 38 30

### CODE REVIEW

File: src/components/researchers/ResearchersListItem.jsx [hunk 1]
```diff
@@ -25,77 +20,93 @@ export const ResearchersListItem = ReactTimeout(class _ResearchersListItem exten
...
+          ${animatingCssClassNames}
+          list-item__cell
+          list-item__cell--title
+        `}>
+          <Link
+            to={link}
+            className='no-select'
+          >
+            <h5 className='title is-size-5 is-monospaced has-text-link'>
+              {this.props.researcher.address}
```
Comment (asselstine) [inline src/components/researchers/ResearchersListItem.jsx]: `<ResearcherNameAndAddress address={this.props.researcher.address} />`

File: src/components/pages/ResearcherPage.jsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    const icon = renderIcon(
+      {
+        seed: address,
+        color: '#dfe',
+        bgcolor: '#fff',
+        size: 10, // width/height of the icon in blocks, default: 10
+        scale: 5 // width/height of each block in pixels, default: 5
+      },
+      canvas
+    )
```
Comment (asselstine) [inline src/components/pages/ResearcherPage.jsx:38]: Let's make sure to confirm a generated icon is appropriate during the meeting tomorrow.  There is a chance we could pull it from one of the Challenge URIs.

**Review by asselstine — APPROVED** [review]
Approve with minor changes.

File: src/projections/projectResearcherVouchedPackages.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+  address = normalizeAddr(address)
+
+  const result = {
+    packages: {}
+  }
+  
+
+  for (let i in events) {
+    // use extraTopics to filter events based on user's eth address!
```
Comment (chuckbergeron) [inline src/projections/projectResearcherVouchedPackages.js:14]: I'm not sure how to do this. I attempted creating a new query with $address as the variable and passing it in to filter the events, but it didn't seem to work. 

I then tried without a var and just hard-coded both of these addresses into the gql:

```
0x8f7f92e0660dd92eca1fad5f285c4dca556e433e
0x8f7F92e0660DD92ecA1faD5F285C4Dca556E433e
```

With no luck in the Apollo Chrome inspector. Will check in with you tomorrow on it! Might be something to do with Apollo Link Ethereum

File: src/components/researchers/ResearchersListItem.jsx (hunk 1 above)
Comment (chuckbergeron) [reply src/components/researchers/ResearchersListItem.jsx]: I didn't feel it was a good idea to show both here with a link to Etherscan. I could see us using the <EnsName /> component here, though!

File: src/projections/projectResearcherVouchedPackages.js (hunk 3 above)
Comment (asselstine) [reply src/projections/projectResearcherVouchedPackages.js:14]: Yeah let's review tomorrow.  I can swing by yours.

**Pull request merged**